.zenodo.json

```diff
@@ -131,11 +129,28 @@
       "name":"Eckert, Carlchristian",
       "type":"Other"
     },
+    {
+      "affiliation":"HZDR, TU Dresden",
+      "name":"Voß, Mika Soren",
+      "type":"Other"
+    },
+    {
+      "affiliation":"HZDR, TU Dresden",
```

Comment:
marco's primary affiliation should probably include LBNL

Reply:
we do not update affiliations, it is the affiliation of the contribution.